fuel/src/main/kotlin/com/github/kittinunf/fuel/core/interceptors/RedirectionInterceptor.kt

```diff
@@ -14,7 +14,7 @@ fun redirectResponseInterceptor(manager: FuelManager) =
                         response.statusCode == HttpsURLConnection.HTTP_MOVED_TEMP ||
                         response.statusCode == 307   // 307 TEMPORARY REDIRECT - https://httpstatuses.com/307
                         ) {
-                    val redirectedUrl = response.headers["Location"]
+                    val redirectedUrl? = response.headers.containsKey("Location") ? response.headers["Location"] : response.headers["location"]
```

Comment:
Do you think if we could use `inline fun String.toUpperCase(): String` or `lowercase` counterpart to simplify our logic a bit?

Reply:
We're almost there! @denniss17 
Let's change a minor thing 👍

Reply:
No, except when we lowercase all keys in the headers map